content/docs/user-guide/troubleshooting.md

```diff
@@ -125,3 +125,52 @@ and retry the DVC command. Specifically, one of:
 
 [internal directories]:
   https://dvc.org/doc/user-guide/project-structure/internal-files
+
+## DVC Experiments may fail in Git shallow clones {#git-shallow}
+
+`dvc exp` commands use internal Git operations which may not work properly in
+[shallow clones](https://git-scm.com/docs/git-clone#Documentation/git-clone.txt---depthltdepthgt).
+Local Git repositories can be unshallowed with `git fetch --unshallow`.
+
+Typically, however, this occurs in transient remote environments such as
+Continuous Integration jobs, which use shallow clones by default. In those
+cases, change their configuration to avoid shallow cloning. Common examples:
+
+<toggle>
+<tab title="CML">
+
+When using [CML](https://cml.dev/doc), repositories can be unshallowed by using:
+
+```cli
+$ cml ci --unshallow
+```
```

Comment:
This ain't working on this repo yet...

https://dvcorg-troubleshootingshallow.gtsb.io/doc/user-guide/troubleshooting?tab=CML#git-shallow

````suggestion
```dvc
$ cml ci --unshallow
```
````

Reply:
resolving since `cli` should work now afaik